driver-embedded/src/main/com/mongodb/embedded/client/MongoDBCAPI.java

```diff
@@ -25,18 +25,21 @@
 
 
     // CHECKSTYLE.OFF: MethodName
+    /**
+     * Interface representing the initialization params for the mongodbcapi library.
+     */
+    interface InitParams {}
```

Comment:
A cheat to represent `libmongodbcapi_init_params` without actually declaring any code inside this interface.